sdk/web3/eth/transaction receipt getter/7d0e9b43-579e-4f9e-8066-801334a9beea.json

```diff
@@ -0,0 +1,47 @@
+{
+  "implementations": {
+    "es6-node": {
+      "dependencies": {
+        "npm": {
+          "web3": {
+            "description": ""
+          }
+        }
+      },
+      "inputFunctions": {
+        "bd95258a-5c14-43b5-b464-6bd778608b05": "const Web3 = require('web3');\r\nconst API_URL_ALCHEMY = 'https://eth-mainnet.alchemyapi.io/v2/71n6GXE_ROLFP-1DELzRWrgsyqqWG2s0';\r\nconst {transaction_hash, provider} = data; \r\n\r\n\r\nvar web3 = new Web3(provider);\r\n\r\nweb3.eth.getTransactionReceipt(transaction_hash)\r\n  .then((tr)=> {\r\n    outputs.transaction_receipt(tr, tag);\r\n  })\r\n  .catch((error)=>{\r\n    outputs.error({error: error}, tag)\r\n})"
```

Comment:
API_URL_ALCHEMY should not be stored here.

Reply:
Sent error should be a of type { "error": string }
Thus the code should rather be like this ```outputs.error({error: error.toString()}, tag)```

Reply:
Thanks Laci! I left it accidentaly.